### DISCUSSION TITLE
Camera name and wifi password

### DISCUSSION THREAD

**Discussion by Tr4v1583** [gopro/labs#622] [Q&A] [answered]
Hi.

is there a possibility to change camera name and wifi password via qr code? Or at least view those on front screen? 
I'd like to buy a go pro 11 with cracked back screen and convert it to a „naked” for fpv drone, but i want to be able to connect via wifi and without back screen it seems to be impossible.
Thanks in advance!

**Comment by dnewman-gpsw** [comment, marked as answer]
If you have labs installed, you can do factory reset (!FRESET + three fast shutter presses) and then do !PA (as a QRCode) to force a new App pairing.   The App can then set new WiFi credentials, do live monitoring etc.

**Reply by Tr4v1583** [reply]
thank You! that is what I was looking for!

**Reply by junebug-junie** [reply]
this ended up bricking my physical camera's UI. I can still run through the app :(

**Reply by dnewman-gpsw** [reply]
That is very unlikely that these commands alone caused any issues.  These commands are just a factory reset, same command as executed from the camera's menu.  Plus this post was about a camera with a damaged rear screen, so what are you trying to do?  The might be a support issue, or least show video of the error you are seeing.
